RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotESProducer.cc

```diff
@@ -51,28 +60,77 @@ OnlineBeamSpotESProducer::OnlineBeamSpotESProducer(const edm::ParameterSet& p) {
 
 void OnlineBeamSpotESProducer::fillDescriptions(edm::ConfigurationDescriptions& desc) {
   edm::ParameterSetDescription dsc;
+  dsc.add<int>("timeThreshold", 48);       // hours
```

Comment:
```suggestion
  dsc.add<int>("timeThreshold", 48)->setComment("hours");
```
this is more practical; this way the comment will appear in the pset help and in the auto-generated configs